### PR TITLE
fix #15798 (yii\grid\CheckboxColumn, yii\grid\RadioButtonColumn conte…

### LINKED ISSUE #15798: yii\grid\CheckboxColumn content option has no effect
I have gone through this [issue ](https://github.com/yiisoft/yii2-bootstrap/issues/168) which does not give any info if it is merged or not to the latest version 

### What steps will reproduce the problem?
```
[ 'class' => \yii\grid\CheckboxColumn::className () ,
   'content' => function($model) {
      return '<div class="custom-control custom-checkbox">' . Html::checkBox ( 'selection' , false , [ 'id' => "'customCheck" . $model->id . "'" , 'class' => 'custom-control-input' ] ) . '<label class="custom-control-label" for="customCheck' . $model->id . '"></label></div>';
    } ,
   'header' => '<div class="custom-control custom-checkbox">' . Html::checkBox ( 'selection_all' , false , [ 'id' => 'customCheck1' , 'type' => 'checkbox'  'class' => 'custom-control-input select-on-check-all' ] ) . '<label class="custom-control-label" for="customCheck1"></label></div>' ,
 ] ,
```
### What is the expected result?
The following HTML is expected inside each cell rendering the checkbox in `GridView` 
```
<td>
    <div class="custom-control custom-checkbox">
        <input name='selection'  checked=false  id= "customCheck1"  class ='custom-control-input' />
        <label class="custom-control-label" for="customCheck1"></label>
    </div>
</td>
```
### What do you get instead?
```
<td>
    <input type="checkbox" name="selection[]" value="1">
</td>
```
### Additional info

| Q                | A
| ---------------- | ---
| Yii version      | 2.0.14
| PHP version      | 5.6
| Operating system |Windows 10

### PR DESCRIPTION
**Pull request by lesha724**
…nt option has no effect)

| Q             | A
| ------------- | ---
| Is bugfix?    | yes
| New feature?  | no
| Breaks BC?    | no
| Tests pass?   | no
| Fixed issues  | #15798

### CODE REVIEW

File: framework/grid/CheckboxColumn.php [hunk 1]
```diff
@@ -113,6 +113,10 @@ protected function renderHeaderCellContent()
      */
     protected function renderDataCellContent($model, $key, $index)
     {
+        if ($this->content !== null) {
+            return call_user_func($this->content, $model, $key, $index, $this);
```
Comment (cebe) [inline framework/grid/CheckboxColumn.php]: better call the parent implementation instead.

File: framework/grid/RadioButtonColumn.php [hunk 2]
```diff
@@ -80,6 +80,10 @@ public function init()
      */
     protected function renderDataCellContent($model, $key, $index)
     {
+        if ($this->content !== null) {
+            return call_user_func($this->content, $model, $key, $index, $this);
```
Comment (cebe) [inline framework/grid/RadioButtonColumn.php]: better call the parent implementation instead.

**Review by cebe — CHANGES_REQUESTED** [review]

**Comment by SilverFire** [comment]
@lesha724 could you add unit tests, please?

There are already tests for [CheckboxColumn](https://github.com/yiisoft/yii2/blob/master/tests/framework/grid/CheckboxColumnTest.php) and [RadioButtonColumn](https://github.com/yiisoft/yii2/blob/master/tests/framework/grid/RadiobuttonColumnTest.php)

**Comment by SilverFire** [comment]
Great! The last thing to do is to add a line in a CHANGELOG.md. Could you do it, please?

**Comment by SilverFire** [comment]
Thank you!

**Pull request merged**